### PR TITLE
Change naming scheme of packet forwarder

### PR DESCRIPTION
**Pull request by jpmeijers**
Preliminary changes for new and preferred TTN Packet Forwarder. More rewriting to come and will be committed to this branch.

### CODE REVIEW

File: _content/gateways/registration.md [hunk 1]
```diff
@@ -4,47 +4,49 @@ title: Gateway Registration
...
-> At the moment, most gateways don't support the Gateway Connector protocol yet. Follow the steps for connecting using the [Via Packet Forwarder](#via-packet-forwarder).
+Please read this guide carefully to understand how to register your gateway with The Things Network.
 
-## Via Gateway Connector
-This is a work in process. The Things Gateway will be the first gateway to use this new protocol. Firmware and guides for other gateways will need to be updated.
+> If you have an off-the-shelf gateway with the default software, it most likely uses the UDP Packet Forwarder. Follow the steps for connecting using the [UDP Packet Forwarder](#via-udp-packet-forwarder).
 
-> Make sure your gateway supports [The Things Network Gateway Connector](https://github.com/TheThingsNetwork/ttn-gateway-connector) before you continue.
+## Via TTN Packet Forwarder
+The Things Gateway uses this packet forwarder and we also have support for Kerlink, Multitech and other Linux based gateways. This is the preferred packet forwarder to use with The Things Network. Firmware and guides for other gateways will need to be updated.
```
Comment (johanstokking) [inline _content/gateways/registration.md]: Actually The Things Gateway uses the gateway connector library 😁

Remove the last sentence, these are our docs and they are leading

File: _content/gateways/registration.md [hunk 2]
```diff
@@ -4,47 +4,49 @@ title: Gateway Registration
 
 # Gateway Registration
 
-We are transitioning to a new way to register and manage your gateways. Please read this guide carefully to understand how it currently works and what will change.
+The are mainly two types of packet forwarders that can be running on your gateway, namely [TTN Packet Forwarder](https://github.com/TheThingsNetwork/packet_forwarder/tree/master) (formerly known as the Gateway Connector) or [UDP Packet Forwarder](https://github.com/TheThingsNetwork/packet_forwarder/tree/legacy) (also known as the Semtech Packet Forwarder or Poly Packet Forwarder).
```
Comment (johanstokking) [inline _content/gateways/registration.md]: The TTN Packet Forwarder is new, it didn't have an old name before

Note that only like 10 users ever used the "gateway connector" option; this is not a well known thing

File: _content/gateways/registration.md [hunk 3]
```diff
@@ -4,47 +4,49 @@ title: Gateway Registration
...
 
-## Via Gateway Connector
-This is a work in process. The Things Gateway will be the first gateway to use this new protocol. Firmware and guides for other gateways will need to be updated.
+> If you have an off-the-shelf gateway with the default software, it most likely uses the UDP Packet Forwarder. Follow the steps for connecting using the [UDP Packet Forwarder](#via-udp-packet-forwarder).
 
-> Make sure your gateway supports [The Things Network Gateway Connector](https://github.com/TheThingsNetwork/ttn-gateway-connector) before you continue.
+## Via TTN Packet Forwarder
+The Things Gateway uses this packet forwarder and we also have support for Kerlink, Multitech and other Linux based gateways. This is the preferred packet forwarder to use with The Things Network. Firmware and guides for other gateways will need to be updated.
+
+> Make sure your gateway runs [The Things Network Packet Forwarder](https://github.com/TheThingsNetwork/packet_forwarder/tree/master) before you continue.
```
Comment (johanstokking) [inline _content/gateways/registration.md]: In fact, it's the other way around. You need to register the gateway first before configuring and running it

File: _content/gateways/registration.md [hunk 4]
```diff
@@ -4,47 +4,49 @@ title: Gateway Registration
...
 
 ### Start with Registration
 You start by [registering your gateway](https://console.thethingsnetwork.org/gateways/register) in the Console.
 
 ![Registration for Gateway Connector](registration-connector.png)
 
 - For **Protocol**, leave **gateway connector** selected.
-- For **Gateway ID**, choose a unique ID of lower case, alphanumeric characters and nonconsecutive `-` and `_`.
-- Select the frequency plan (determined by your region) the gateway uses.
+- For **Gateway ID**, choose a unique ID of lower case, alphanumeric characters and nonconsecutive `-` and `_`. **DO NOT** use an EUI.
```
Comment (johanstokking) [inline _content/gateways/registration.md]: EUIs are not bad

**Review by johanstokking — CHANGES_REQUESTED** [review]

**Comment by jpmeijers** [comment]
Changes applied. Ready for another review.

Please only merge. Leave this branch in existence so that I can continue the work when the console is updated.

**Review by johanstokking — APPROVED** [review]

**Pull request merged**